doc/articles/uno-development/docfx.md

```diff
@@ -123,5 +123,5 @@ To run the script on Windows:
 
 1. Make sure `crosstargeting_override.props` is not defining UnoTargetFrameworkOverride
 2. Open a Developer Command Prompt for Visual Studio (2019 or 2022)
-3. Go the the uno\build folder (not the uno\src\build folder)
+3. Go the uno\build folder (not the uno\src\build folder)
```

Comment:
```suggestion
3. Go to the uno\build folder (not the uno\src\build folder)
```

Reply:
Change will be done on this follow up PR: https://github.com/unoplatform/uno/pull/15144